### DISCUSSION TITLE
[V23] 无法保存系统设置

### DISCUSSION THREAD

**Discussion by sany98215** [linuxdeepin/developer-center#5040] [Troubleshooting | 故障排查] [answered]
当前版本为最近发布的内侧版本（非日更），最近几个版本遇到一个问题：设置中心设置无法保存，比如修改鼠标配置-自然滚动，设置后，本次登陆生效，重启后失效。这个问题在我的本本里一直没有解决，没有找到对应的bug,也没有看到修复。

**Comment by Decodetalkers** [comment]
看错了，哦，我明天看看

**Comment by sany98215** [comment, marked as answer]
我发现问题了：系统分区加载顺序中/home分区有文件依赖了另外一个分区，因此启动后不清楚影响到哪里，导致保存失败。我修改了分区加载顺序，现在可以正常保存
